common/test/acceptance/pages/studio/component_editor.py

```diff
@@ -108,43 +109,83 @@ class ComponentVisibilityEditorView(BaseComponentEditorView):
     """
     A :class:`.PageObject` representing the rendered view of a component visibility editor.
     """
-    OPTION_SELECTOR = '.modal-section-content .field'
+    OPTION_SELECTOR = '.partition-group-control .field'
+    ALL_LEARNERS_AND_STAFF = 'All Learners and Staff'
```

Comment:
This is a page object for bok choy tests-- see the file path. So no, they do not need to be translated.